src/components/listItems/index.js

```diff
@@ -93,17 +92,17 @@ export const UserListItem = ({
       ? user.contextPermissions.reputation
       : 0
     : user.reputation && typeof user.reputation === 'number'
-      ? user.reputation
-      : user.totalReputation && typeof user.totalReputation === 'number'
-        ? user.totalReputation
-        : 0;
+    ? user.reputation
+    : user.totalReputation && typeof user.totalReputation === 'number'
+    ? user.totalReputation
```

Comment:
Did we intentionally lose the tabs here? The indentation in nested ternaries is super helpful for following the logic (although I know we shouldn't use nested ternaries in the first place :P)

Reply:
Noticed ~6 more instances of losing whitespace in nested ternaries. Is this eslint or something manual you did?

Reply:
I have my own prettier config setup in vscode but Prettier should have taken care of any oddities when I committed yeah? What can I do? @brianlovin

Reply:
No clue - maybe our commit hook is reading your prettier config? cc @mxstbr for oddities here - any idea what could cause this bad ternary formatting?

Reply:
@brianlovin do you guys use Tabs or spaces? I saw this in the prettier docs `Tabs will be used for indentation but Prettier uses spaces to align things, such as in ternaries`

Sorry for the late reply, I've been on vacation

Reply:
Take a look at this; https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAHAThVBnAdNuAGzjHgBMAdKAAlroH5qMs8YALOAWzn1RIEsAhoVwB3QdjZU6dJE0w5ccdJnTSZDeS1zsuPceihiJUmhtpzmi3d1wAjAOa4ycAGaCAroRggANCCwYfmhsZFBBFQhRAAUIhFCUQQA3CH4yPxA7dEEwAGs4GABlVBz+KAdkGHQPOH82GE5CAHU2fnhsErA4Qvi2-iS2gE9kcGxQ-zKCdBho7IdOQWR3QgJ-ACtsAA8AIWy8gsLBbgAZMrgl4VWQDc3CsodiAEUPCHgLldq0CKmRu0E7QaEaAZDBlGBNNLsZAADgADP5mAQmtlUCMMHApklzv50HAAI4efi42aCeaLJDLK4ETj8SrVT7Ye5PF5vCmXT4wf4QshQpAAJn8VUE-EI9wAwhBOAsRlBoNiQB4CAAVf4JSlwAC+GqAA